src/components/ProductForm/ProductForm.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+const ProductForm = ({
+  name,
+  id,
+  image,
+  price,
+  rate,
+  disPrice,
+  gender,
+  status,
+}) => {
+  const [isHeart, setIsHeart] = useState(true);
+
+  const heartColorChange = () => {
+    setIsHeart(!isHeart);
+  };
```

Comment:
```suggestion
  const heartColorChange = () => {
    setIsHeart(isHeart => !isHeart);
  };
```
이전의 상태값을 활용할 때는 함수형 업데이트를 활용해주세요!

Reply:
감사합니다 도현님 수정 헀습니다.